src/app/components/mno-select-app-wizard/mno-select-app-wizard.directive.coffee

```diff
@@ -0,0 +1,82 @@
+
+angular.module 'mnoEnterpriseAngular'
+  .directive('mnoSelectAppWizard', ->
+    return {
+      restrict: 'EA'
+      templateUrl: 'app/components/mno-select-app-wizard/mno-select-app-wizard.html',
+      scope: {
+        appsData: '='
+        isPanelShown: '='
+      }
+
+      controller: ($scope, $window, MnoeMarketplace, $sce, $uibModal) ->
+        $scope.selectedCategory = ''
+        $scope.searchTerm = ''
+
+        $scope.trustSrc = (src) ->
+          $sce.trustAsResourceUrl(src)
```

Comment:
I think you should trust the logo directly in the controller and not publish the method in the template.